rdev/R/utils.R

```diff
@@ -196,41 +196,60 @@ expand.ev <- function(...) {
 is.numeric.data.frame <- function(x) sapply(x, is.numeric)
 
 
+
+##' Render an \code{mrgsolve} model.
+##' 
+##' @param x model object
+##' @param file file name to use for building; should have \code{.Rmd} extension
+##' @param quiet not used
+##' @param ... not used
+##' 
 render <- function(x,file=tempfile(fileext=".Rmd"),quiet=TRUE,...) {
   stopifnot(requireNamespace("rmarkdown"))
-  bl <- modelparse(x@code,warn=FALSE,split=FALSE,drop_blank=FALSE,comment_re="//+")
+  bl <- modelparse(x@code,warn=FALSE,split=FALSE,drop_blank=FALSE,comment_re="//")
   txt <- bl[["PROB"]]
   if(is.null(txt)) stop("Couldn't find $PROB in model.",call.=FALSE)
   
   bl[["PROB"]] <- NULL
   anot <- tempfile(fileext=".RDS")
   
   for(i in seq_along(bl)) {
+    bl[[i]] <- bl[[i]][bl[[i]] !=""]
     bl[[i]] <- paste(bl[[i]], collapse="\n") 
   }
-  saveRDS(file=anot, list(details=details(x),code=x@code,bl=bl))
+
+
+  w <- options()$width
+  options(width=120)
```

Comment:
is this code to not truncate the printed table when capturing the output or something?